packages/salesforcedx-vscode-core/test/vscode-integration/commands/util/postconditionCheckers.test.ts

```diff
@@ -440,6 +444,10 @@ describe('Postcondition Checkers', () => {
       settingsStub = env.stub(sfdxCoreSettings, 'getConflictDetectionEnabled');
       conflictViewStub = env.stub(conflictView, 'visualizeDifferences');
       appendLineStub = env.stub(channelService, 'appendLine');
+      env.stub(WorkspaceContext, 'getInstance').returns(mockWorkspaceContext);
```

Comment:
Not that I want to bloat this file any more but... is it worth adding a test where the OrgType is SourceTracked?

Reply:
I don't really want to add to the integration test.  I added just enough here to get it to pass.  When this integ test is converted to a unit test, that scenario could be added there.